Improve signer related docs (#215)
software-mansion/starknet.py: docs/guide.rst

    :dedent: 4

Using different signing methods
-------------------------------

> By default, :ref:`Account Client` uses signing method reflecting this required by
> OpenZeppeling account contract. If for any reason you want to use a different

Reviewer: ```suggestion
By default, :ref:`Account Client` uses signing method of OpenZeppelin's account contract. If for any reason you want to use a different
```
